### PR TITLE
22286 update to use new furnishing groups approach

### PR DESCRIPTION
**Pull request by kzdev420**
*Issue #:* /bcgov/entity#22286

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/src/legal_api/models/furnishing.py [hunk 1]
```diff
@@ -125,15 +124,14 @@ def find_by(cls,  # pylint: disable=too-many-arguments
...
 
     @classmethod
-    def get_next_grouping_identifier(cls):
-        """Return the next grouping_identifier from the sequence."""
-        return db.session.execute("SELECT nextval('grouping_identifier')").scalar()
+    def get_furnishing_group_id(cls):
+        """Return thefurnishing_group_id."""
+        furnishing_group = FurnishingGroup()
+        furnishing_group.save()
+        return furnishing_group.id
```
Comment (kzdev420) [inline legal-api/src/legal_api/models/furnishing.py:137]: I think there will be more processes like creating the XML_Payload, and others too.
So, I'd like to keep this function for now.

**Review by leodube-aot — APPROVED** [review]
LGTM

File: legal-api/src/legal_api/models/furnishing.py [hunk 2]
```diff
@@ -125,15 +124,14 @@ def find_by(cls,  # pylint: disable=too-many-arguments
...
         if furnishing_group_id:
             query = query.filter(Furnishing.furnishing_group_id == furnishing_group_id)
 
         return query.all()
 
     @classmethod
-    def get_next_grouping_identifier(cls):
-        """Return the next grouping_identifier from the sequence."""
-        return db.session.execute("SELECT nextval('grouping_identifier')").scalar()
+    def get_furnishing_group_id(cls):
```
Comment (argush3) [inline legal-api/src/legal_api/models/furnishing.py:133]: can we name it `get_next_furnishing_group_id`?

Comment (kzdev420) [reply legal-api/src/legal_api/models/furnishing.py]: Updated

**Pull request merged**